examples/cool_features_demo.py

```
@@ -0,0 +1,319 @@
+"""
+Demo of the cool new features in structlite:
+1. Async Validation
+2. Struct Builders (Fluent Interface)
+3. Database Integration
+4. Field Transformers
+"""
+
+import asyncio
+import sqlite3
+from datetime import datetime, date
+from typing import Annotated, Optional
```

Ruff (0.12.2)

`typing.Annotated` imported but unused

Remove unused import: `typing.Annotated`

(F401)